src/shared_gui_components/OSCellWrapper.cpp

```diff
@@ -63,13 +64,15 @@ OSCellWrapper::OSCellWrapper(OSGridView* gridView, QSharedPointer<BaseConcept> b
   m_layout->setSpacing(0);
   m_layout->setVerticalSpacing(0);
   m_layout->setHorizontalSpacing(0);
-  m_layout->setContentsMargins(0, 0, 1, 1);
   this->setLayout(m_layout);
   this->setAttribute(Qt::WA_StyledBackground);
   this->setObjectName("OSCellWrapper");
-  setStyleSheet("QWidget#OSCellWrapper { border: none; border-right: 1px solid gray; border-bottom: 1px solid gray; }"
-                "QWidget#OSCellWrapper[header=\"true\"]{ border: none; border-top: 1px solid black; border-right: 1px solid gray; border-bottom: 1px "
-                "solid black; }");
+  setStyleSheet(
```

Comment:
@antonszilasi These border lines in the stylesheet were what remained when we hid the cells